3rdWeek/특이한 정렬.js

```diff
@@ -0,0 +1,14 @@
+function solution(numlist, n) {
+    numlist.sort((a, b) => {
+        if (Math.abs(a - n) > Math.abs(b - n)){
+           return 1;
+        }
+        else if (Math.abs(a - n) < Math.abs(b - n)) {
+            return -1;
+        } else {
+            if (a > b) return -1;
+            else return 1;
+        }
```

Comment:
-1을 반환해야 하는 조건과 1을 반환해야 하는 조건을 정리해서 조건식을 줄일 수 있을 것 같습니다

Reply:
@rookedsysc 님의 피드백을 바탕으로 아래와 같이 작성해보았는데요!  
```suggestion
        if (Math.abs(a - n) > Math.abs(b - n) || (Math.abs(a - n) === Math.abs(b - n) && a <= b){
           return 1;
        }
         else if (Math.abs(a - n) < Math.abs(b - n) || (Math.abs(a - n) === Math.abs(b - n) && a > b){
           return -1;
        }
```
이렇게 정렬기준이 복잡하지 않은 경우에는 위 같이정리하는게 좋다고 생각합니다! 다만 `정렬기준이 복잡할 경우` 아래와 같이 로직을 짜서 기준을 잡는게 편하더라고요! 
- a가 b보다 큰경우
- b가 a보다 큰경우
- a와 b가 같은경우

복잡하지 않은경우는 가독성을 위해 정리하는게 좋겠네요 :) 피드백 감사합니다

Reply:
저는 이런 방식이 좋다고 생각했어요.

```js
if (Math.abs(a - n) > Math.abs(b - n) {
           return 1;
}

elif (Math.abs(a - n) === Math.abs(b - n) && a <= b){
           return 1;
}

return -1;
```